services/graphite/service.go

```diff
@@ -219,22 +209,42 @@ func (s *Service) openUDPServer() (net.Addr, error) {
 		for {
 			n, _, err := conn.ReadFromUDP(buf)
 			if err != nil {
-				batcher.Flush()
 				conn.Close()
 				return
 			}
 			for _, line := range strings.Split(string(buf[:n]), "\n") {
-				point, err := s.parser.Parse(line)
-				if err != nil {
-					continue
-				}
-				batcher.In() <- point
+				s.handleLine(line)
 			}
 		}
 	}()
 	return conn.LocalAddr(), nil
 }
 
+func (s *Service) handleLine(line string) {
+	if line == "" {
+		return
+	}
+	// Parse it.
+	point, err := s.parser.Parse(line)
+	if err != nil {
+		s.logger.Printf("unable to parse line: %s", err)
+		return
+	}
+
+	f, ok := point.Fields()["value"].(float64)
+	if !ok {
```

Comment:
Good idea.  I'm thinking of adding a `write-tracing` option as well and only log these if that is enabled.  Different PR though.